pkg/network/tracer/tracer_classification_test.go

```diff
@@ -537,6 +539,139 @@ func testProtocolClassification(t *testing.T, cfg *config.Config, clientHost, ta
 			teardown:   defaultTeardown,
 			validation: validateProtocolConnection,
 		},
+		{
```

Comment:
what about some cases to ensure we don't identify false-positives?

Reply:
do you have an example? I can create random buffers but that's not good enough
the real scenario we wished to check here is multiple different protocols (even not supported protocols) and ensure we do not classify them, but it wont guarantee success